tfe/resource_tfe_organization_module_sharing.go

```diff
@@ -11,10 +11,11 @@ import (
 
 func resourceTFEOrganizationModuleSharing() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceTFEOrganizationModuleSharingCreate,
-		Read:   resourceTFEOrganizationModuleSharingRead,
-		Update: resourceTFEOrganizationModuleSharingUpdate,
-		Delete: resourceTFEOrganizationModuleSharingDelete,
+		DeprecationMessage: "the tfe_organization_module_sharing resource is deprecated, please use tfe_admin_organization_settings instead",
```

Comment:
💯

Reply:
Created a test to verify this deprecation warning (verified!) ✅ 

![Screen Shot 2022-12-05 at 5 14 31 PM](https://user-images.githubusercontent.com/3526523/205774508-462ced80-4641-4adf-ac56-db993d09824d.png)